apps/api/Dockerfile

```
@@ -1,25 +1,28 @@
 RUN apk add --no-cache libc6-compat openssl && apk update
 WORKDIR /app
-RUN npm install -g turbo@^1.12.3
+RUN npm install -g turbo@${TURBO_VERSION}
 COPY . .
 
 # Prune the workspace to only include nextjs and its dependencies
 RUN turbo prune f3-nation-api --docker
 
 FROM --platform=linux/amd64 node:24-alpine AS installer
+ARG TURBO_VERSION
 RUN apk add --no-cache libc6-compat openssl && apk update
```

Hadolint (2.14.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)